rebar.lock

```diff
@@ -1,4 +1 @@
-[{<<"lfe">>,
-  {git,"https://github.com/rvirding/lfe.git",
-       {ref,"cd96efc3d2a73ecff7f3e2ef3563b73b098e4844"}},
-  0}].
+[{<<"lfe">>,{pkg,<<"lfe">>,<<"1.0.2">>},0}].
```

Comment:
We can't do this. If you don't want to use Hex, then we can't at all, unless we do some profile voodoo or something. If you're opposed to Hex support, I guess I could maintain a branch that's identical except for the lfe dep spec.